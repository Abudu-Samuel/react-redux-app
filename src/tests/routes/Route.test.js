/* eslint-disable */
import Routes from '../../route';

describe('<Routes />', () => {
  it('render routes in the app', () => {
    const wrap = shallow(<Routes />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});