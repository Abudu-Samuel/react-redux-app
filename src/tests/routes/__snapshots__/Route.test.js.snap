// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> render routes in the app 1`] = `
<BrowserRouter>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/about"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/courses"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/course"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/course/:id"
    />
    <Route
      component={[Function]}
      exact={true}
    />
  </Switch>
</BrowserRouter>
`;
